lib/model/bank_account.dart

```diff
@@ -10,7 +11,7 @@ class BankAccountFields extends BaseEntityFields {
   static String name = 'name';
   static String symbol = 'symbol';
   static String color = 'color';
-  static String value = 'value';
+  static String starting_value = 'starting_value';
```

Comment:
Is better to use camel case for variable names: "The variable name 'starting_value' isn't a lowerCamelCase identifier.
Try changing the name to follow the lowerCamelCase style." [https://dart-lang.github.io/linter/lints/non_constant_identifier_names.html](url) . This should be done in every file it is used.

Reply:
Damn... you're right. I will